index.js

```diff
@@ -93,10 +85,32 @@ Jstify.prototype._flush = function (next) {
   next();
 };
 
+function parseExtensions (extensions) {
+  if (typeof extensions === 'string') {
+    return extensions.replace(' ', '').split(',');
```

Comment:
Nah, no string parsing. Take the values as they come. But you need to do the `opts.extensions._` bit because that's how [subarg](https://github.com/substack/subarg) handles "contexts":

```js
// -t [ jstify --extensions .html  ]
opts = {
  extensions: '.html',
}

// -t [ jstify --extensions .html --extensions .svg ]
opts = {
  extensions: [ '.html', '.svg' ]
}

// -t [ jstify --extensions [ .html ] ]
opts = {
  extensions: {
    _: [ '.html'  ]
  }
}

// -t [ jstify --extensions [ .html .svg ] ]
opts = {
  extensions: {
    _: [ '.html', '.svg' ]
  }
}

// DON'T WORRY ABOUT THIS CASE:
// -t [ jstify --extensions [ .html .svg ] --extensions .tpl ]
opts = {
  extensions: [
    { _: [ '.html', '.svg' ] },
    '.tpl'
  ]
}
```

The easier thing to do is:

```
var extensions = [].concat(opts.extensions, opts.extensions._).filter(Boolean);
```

That'll handle all the cases but the last - don't worry about that one.